Sources/CreateOptions/ConfigOptions.swift

```diff
@@ -712,6 +712,10 @@ public struct ConfigOptions: ParsableConfiguration {
         /// Rename anonymous collection elements. By default, use a singularized form of the property name
         @Option public var collectionElements: [String: String] = [:]
     }
+
+    /// If `true`, CreateAPI generates request body structures for "multipart/form-data" format.
+    /// Otherwise such request body fallbacks to `Data` type. The default value is `false`.
```

Comment:
So these comments are used to updated documentation too, if you run `make documentation`, it should update **Docs/ConfigOptions.md** with the definition and this content 🙏 

While we are at it, I think we could add some more context to help describe the purpose of this option... How about something like this:

```suggestion
    /// If `true`, CreateAPI generates request body structures for "multipart/form-data" format just like it would for "application/json".
    ///
    /// Otherwise the `body` of the generated `Request` will fallback to `Data`. The default value is `false`.
    ///
    /// When using Get and it's `APIClient`, because Multipart Form Data isn't supported from the `Request` `body` property, it is best to leave this option set to `false`. 
    /// If however you have implemented your API Client, and you prefer to use structured `Codable` types to encode a Multipart Form Data request body, setting this value to `true` can be more convenient. 
    /// You might also need to use the [`dataTypes`](#datatypes) option to customise the type used to represent `binary` data.
```

Reply:
Adopted this as well as flipping true and false for new option name